src/components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+/**
+ * Displays user profile avatar image with/withour user name
+ */
+const Avatar = ({ className, size, userName, imgUrl }) => {
+  const avatarClass = classes(className, 'Avatar');
+  const avatarSize = size >= 20 && size <= 200 ? size : 40;
+  const avatarLetterSize =
+    size >= 20 && size <= 200
+      ? { height: size, width: size, fontSize: size * 0.8 }
```

Comment:
where is the `* 0.8` coming from ?

Reply:
So the initial letter inside the circle is slightly smaller than the circle. 
![image](https://user-images.githubusercontent.com/32206137/55979106-6b199d00-5c5f-11e9-94a6-92c48295ddc2.png)